lms/static/js/utils/dropdown.js

```diff
@@ -0,0 +1,235 @@
+var edx = edx || {},
```

Comment:
I've never written a Jasmine test, but would love to learn how. You think you can help me with this?